src/DETHRACE/common/opponent.c

```diff
@@ -254,8 +254,6 @@ void ProcessCurrentObjective(tOpponent_spec* pOpponent_spec, tProcess_objective_
     case eOOT_wait_for_some_hapless_sod:
         ProcessWaitForSomeHaplessSod(pOpponent_spec, pCommand);
         break;
-    case eOOT_rematerialise:
-        break;
```

Comment:
Can these be surrounded with `#ifdef DETHRACE_FIX_BUGS`?
My compiler emits a warning when a switch is missing a enum value.